autosubmit/autosubmit.py

```diff
@@ -2862,9 +2860,11 @@ def statistics(expid, filter_type, filter_period, file_format, section_summary,
                     Log.info("Plotting stats...")
                     monitor_exp = Monitor()
                     # noinspection PyTypeChecker
-                    monitor_exp.generate_output_stats(expid, jobs, file_format, section_summary, jobs_summary, not hide, period_ini, period_fi, not hide,
-                                                      queue_time_fixes)
-                    Log.result("Stats plot ready")
+                    report_created = monitor_exp.generate_output_stats(expid, jobs, file_format, hide, section_summary,
+                                                                       jobs_summary, period_ini, period_fi,
+                                                                       queue_time_fixes)
+                    report_message = "Statistics plot ready" if report_created else "No statistics plot produced."
```

Comment:
Previously, the return value of `generate_output_stats` was being ignored, and the command always printed "Stats plot ready", even when you didn't have any plot created. Now it says whether plots were created or not (since the function was already returning a `bool`...)